### DISCUSSION TITLE
How to handle distortion images?

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2933] [Q&A] [answered]
While I am testing, I got images as below
Please let me know how to fix it.
![img_7177_720](https://github.com/lllyasviel/Fooocus/assets/91205658/5a33669e-eeca-4347-9986-4d67bcf0fc37)

**Comment by ILikeToasters** [comment, marked as answer]
If you don't post clear questions it is difficult for people to answer them. For example with this post you do not describe what you feel needs fixing with the image. Also including things like prompts that gave you the results will help since it will require less back and forth when people try to figure out what you are asking.

**Reply by oldhand7** [reply]
I think some of the generated images are distorted especially eye, hand, limbs are distorted. Those can be fixed by inpaint after generated. But I have no idea of how to implement them. Please let me know how to do it. Thank you.
